k8s/sandbox/cnp/cnp-plum-recipes-service.rc.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: plum-recipe-backend-rc
+  namespace: cnp
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:aat-rc-*
+spec:
+  releaseName: plum-recipe-backend-rc
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: plum-recipe-backend
+    version: 0.0.2
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "2700c28a-aa8c-4669-8718-1e4239dee797"
+      image: hmcts.azurecr.io/hmcts/plum-recipe-backend:aat-rc-301f251eab1904d382467433160c23db77fb1ad3
+      ingressHost: plum-recipe-backend.service.core-compute-aat-rc.internal
+      applicationPort: 4550
+      environment:
+        IDAM_OAUTH2_TOKEN_ENDPOINT: https://preprod-idamapi.reform.hmcts.net:3511/oauth2/token
```

Comment:
this won't work in sandbox

does plum actually use this though?

Reply:
No, it was just to add some test env vars